<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tpPlugin</class>
 <widget class="QMainWindow" name="tpPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thomson Parabola</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tracks</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1" columnstretch="1,1">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Ion species</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
           <item row="1" column="0">
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Charge number Z</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Mass number A</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="Z_sb">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="A_sb">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Simulation Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10" columnstretch="0,1">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Init E</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="initE_sb">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> MeV</string>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>End E</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="endE_sb">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="suffix">
              <string> MeV</string>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="escale_cb">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>scale to protons</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QSpinBox" name="points_sb">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="neutrinoSave" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="addTrack_pb">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Add track</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Points</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QTableWidget" name="trackTable">
          <property name="columnCount">
           <number>6</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>A</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Z</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>init E</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>end E</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>n Points</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Simulation Box</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Vertex 1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QVecInput" name="boxV1">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>32767</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Vertex 2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QVecInput" name="boxV2">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>B field</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Center</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QVecInput" name="Bcenter">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Box</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QVecInput" name="Bbox">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Vec</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QVecInput" name="Bvec">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>E field</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Center</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QVecInput" name="Ecenter">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Box</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QVecInput" name="Ebox">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Vec</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QVecInput" name="Evec">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="neutrinoSave" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="drawTab">
       <attribute name="title">
        <string>Outlook</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridGl"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad_config">
   <property name="text">
    <string>&amp;Load config</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVecInput</class>
   <extends>QLineEdit</extends>
   <header>qvecinput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>boxV1</tabstop>
  <tabstop>boxV2</tabstop>
  <tabstop>Bcenter</tabstop>
  <tabstop>Bbox</tabstop>
  <tabstop>Bvec</tabstop>
  <tabstop>Ecenter</tabstop>
  <tabstop>Ebox</tabstop>
  <tabstop>Evec</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
